source/Core/Autoload/UnifiedNameSpaceClassMap.php

```diff
@@ -2853,12 +2853,6 @@
         'isInterface'      => false,
         'isDeprecated'     => false
     ],
-    'OxidEsales\Eshop\Core\MailValidator'                                        => [
```

Comment:
We could deprecate it (isDeprecated -> false), but not remove the section, i guess.

Reply:
This class does not exist anymore in the pull request, but we could deprecate it in #723, what do you think? #723 should target `b-6.x`, so we can deprecate the class there and then finally remove it with this pull request that targets `master`